dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/RelationshipRestRepository.java

```diff
@@ -60,4 +94,100 @@ public RelationshipRest findOne(Context context, Integer integer) {
     public DSpaceResource<RelationshipRest> wrapResource(RelationshipRest model, String... rels) {
         return new RelationshipResource(model, utils, rels);
     }
+
+    protected RelationshipRest createAndReturn(Context context)
+        throws AuthorizeException, SQLException, RepositoryMethodNotImplementedException {
+
+        HttpServletRequest req = getRequestService().getCurrentRequest().getHttpServletRequest();
+        ObjectMapper mapper = new ObjectMapper();
+        RelationshipRest relationshipRest = null;
+        try {
+            relationshipRest = mapper.readValue(req.getInputStream(), RelationshipRest.class);
+        } catch (IOException e1) {
+            throw new UnprocessableEntityException("error parsing the body");
+        }
+
+        Relationship relationship = new Relationship();
+        Item leftItem = itemService.find(context, UUIDUtils.fromString(req.getParameter("leftItem")));
+        Item rightItem = itemService.find(context, UUIDUtils.fromString(req.getParameter("rightItem")));
+        RelationshipType relationshipType = relationshipTypeService
+            .find(context, Integer.parseInt(req.getParameter("relationshipType")));
```

Comment:
Again, here it looks like we are creating (POST) based on params.  These should likely be passed in via the request body.

Also, where is `createAndReturn()` used?  I don't see usage anywhere?

Reply:
This will be handled with 
https://github.com/DSpace/DSpace/pull/2332#discussion_r249522435